### ISSUE TITLE
Feature Request: Start minimized to system tray in Flatpak package

### ISSUE THREAD

**Issue by andresgarcia0313** [wwmm/easyeffects#3207]
### EasyEffects Version

7.1.6

### What package are you using?

Flatpak (Flathub)

### Distribution

All

### Describe the bug

Currently, the Flatpak package of EasyEffects does not offer the option to automatically start minimized to the system tray nor to minimize to the system tray when the window is minimized. This causes the application to occupy space in the taskbar, which can be inconvenient for users who prefer to keep their taskbar uncluttered.

### Expected Behavior

The expected behavior is for EasyEffects to offer an option to:

1. Start automatically minimized to the system tray on startup.
2. Minimize to the system tray instead of the taskbar when the user chooses to minimize the application.

### Debug Log

<details><summary>Debug Log</summary><br/>

```
Paste your log here
```
</details>

### Additional Information

This functionality will enhance the user experience, allowing for a cleaner and more organized workspace. Additionally, it will align EasyEffects with the expected behavior of many modern applications that support minimizing to the system tray.

Thank you for your attention and consideration in implementing this improvement, which will undoubtedly benefit many users. 😊🎉🚀

**Comment by violetmage** [comment]
1. Does the option `Settings -> General -> Launch Service at System Startup` not work?

2. Easyeffects does not use a system tray icon, even though it can run in the background without an open window.

**Comment by wwmm** [comment]
Gtk4 does not support system tray by design. So untill the day we move away from gtk comes adding a tray icon won't be an option.

That being said it should be able to start with its window hidden.

**Comment by andresgarcia0313** [comment]
It's necessary to restart to activate and validate the service, and with this, it works without issues. However, it's annoying to have to hide the icon in the tray every time I restart the system using KDock. Who knows how kdock does it with gtk4.

**Comment by Skynet011** [comment]
Fedora 41 Gnome here.
In my fedora 40 setup the app would immediately go to background.
In Fedora 41 that does not happen. It opens the windows everytime at boot. It would be cool if there was a way to make it go to background again. :)

**Comment by wwmm** [comment]
> Fedora 41 Gnome here. In my fedora 40 setup the app would immediately go to background. In Fedora 41 that does not happen. It opens the windows everytime at boot. It would be cool if there was a way to make it go to background again. :)

We did not make any change related to this. It still does to the background just fine on my computer while using a native package. The Flatpak package that for some reason I do not remember sometimes fails to do so. As far as I remember what should be done on our side was done.

@Skynet011 how did you enable the autostart on boot? Based on your  report it is almost like an autostart file without the option `--gapplication-service` was made.

**Comment by Skynet011** [comment]
You're right. Now that i added the --gapplication-service in the autostart it works just as before.
I tried many ways but no luck.
With flatpaks i'm still getting to know a bit more about how to configure them if i need to.

**Comment by wwmm** [comment]
> You're right. Now that i added the --gapplication-service in the autostart it works just as before. I tried many ways but no luck. With flatpaks i'm still getting to know a bit more about how to configure them if i need to.

A long time ago Flatpak users had to create a custom autostart file manually. But as far as I remember this should not be the case anymore. Flatpak runtime or libportal runtime should have shown a window where permission for the autostart was asked for.

**Comment by brunetton** [comment]
I didn't tried this but it looks promising: https://github.com/boomshop/easyeffectstray

(I'm aware this comment looks like Aliexpress products reviews 😆)

**Comment by wwmm** [comment]
The Qt branch has a tray icon and a menu that will be improved as time goes by. I am not sure about how well the interaction with Flatpak will be but this would be a different issue. Let's close this one.

**Comment by andresgarcia0313** [comment]
Thank you validate the system tray icon in kubutnu lts and what I have tried is spectacular and this super options thank you very much for sharing and I see that you are already achieving that the project in addition to being opensource is profitable for you for which I congratulate you
